### PR TITLE
Correct RGL Unity Plugin docs

### PR DESCRIPTION
**Pull request by msz-rai**

### CODE REVIEW

File: docs/DeveloperGuide/Tutorials/AddANewLiDARModel/index.md [hunk 1]
```diff
@@ -1,23 +1,29 @@
...
+    Not all lidar types are supported by `RGL`. Unfortunately, in the case of `MEMs` *LiDARs*, there is a non-repetitive phenomenon - for this reason, the current implementation is not able to reproduce their work.
+
 ## 1. Add new LiDAR model
-The example shows the addition of a *LiDAR* named `NewLidarModel` using the `VelodyneVLP16` parameters.
+The example shows the addition of a *LiDAR* named `NewLidarModel`.
 
 To add a new *LiDAR* model, perform the following steps:
 
-1. Add its name to the `LidarModels.cs`
+1. Add its name to the `LidarModels.cs` at the end of the enumeration. The order of enums must not be changed to keep existing prefabs working.
```
Comment (prybicki) [inline docs/DeveloperGuide/Tutorials/AddANewLiDARModel/index.md:12]: Add space to display screenshot in separate paragraph.
```suggestion
1. Add its name to the `LidarModels.cs` at the end of the enumeration. The order of enums must not be changed to keep existing prefabs working.

```

File: docs/DeveloperGuide/Tutorials/AddANewLiDARModel/index.md [hunk 2]
```diff
@@ -1,23 +1,29 @@
...
 ![lidar_models](lidar_models.png)
 
 1. If the *LiDAR* has a non-uniform laser array construction (e.g. different linear / angular spacing between lasers), add an entry to the `LaserArrayLibrary`, otherwise, skip this step.
+   
+    !!! warning "Coordinate system"
+        Keep in mind that *Unity* has a left-handed coordinate system, while most of the lidar's manuals use a right-handed coordinate system. In that case, reverse sign of the values of the angles.
 ![lidar_array](lidar_array.png)
```
Comment (prybicki) [inline docs/DeveloperGuide/Tutorials/AddANewLiDARModel/index.md:19]:
```suggestion
        Keep in mind that *Unity* has a left-handed coordinate system, while most of the lidar's manuals use a right-handed coordinate system. In that case, reverse sign of the values of the angles.

![lidar_array](lidar_array.png)
```

File: docs/DeveloperGuide/Tutorials/AddANewLiDARModel/index.md [hunk 3]
```diff
@@ -1,23 +1,29 @@
...
 
+
+
 1. Add an entry to `LidarConfigurationLibrary`. If the *LiDAR* has a uniform laser generate a uniform one using static method `LaserArray.Uniform()` - just like the `RangeMeter`.
 ![lidar_configuration](lidar_configuration.png)
 
-1. Done. New LiDAR preset should be available via Unity Inspector.
+1. Done. New *LiDAR* preset should be available via *Unity Inspector*.
 ![done](done.png)
```
Comment (prybicki) [inline docs/DeveloperGuide/Tutorials/AddANewLiDARModel/index.md:27]:
```suggestion
1. Done. New *LiDAR* preset should be available via *Unity Inspector*.

![done](done.png)
```

**Review by prybicki — APPROVED** [review]

File: docs/DeveloperGuide/Tutorials/AddANewLiDARModel/index.md (hunk 1 above)
Comment (dmoszynski) [reply docs/DeveloperGuide/Tutorials/AddANewLiDARModel/index.md:12]: In my opinion, it looks better without change.

Before:
Screenshot from 2023-06-22 13-53-39

Change
image

File: docs/DeveloperGuide/Tutorials/AddANewLiDARModel/index.md (hunk 2 above)
Comment (dmoszynski) [reply docs/DeveloperGuide/Tutorials/AddANewLiDARModel/index.md:19]: Same as before.

File: docs/DeveloperGuide/Tutorials/AddANewLiDARModel/index.md (hunk 3 above)
Comment (dmoszynski) [reply docs/DeveloperGuide/Tutorials/AddANewLiDARModel/index.md:27]: Same as before.

**Pull request merged**